src/Utilities/WellKnownTypes.cs

```diff
@@ -746,6 +558,16 @@ void AddIfNotNull(INamedTypeSymbol type)
             }
         }
 
+        public static INamedTypeSymbol SystemSecurityCryptographyCipherMode(Compilation compilation)
+        {
+            return compilation.GetTypeByMetadataName(WellKnownTypeNames.SystemSecurityCryptographyCipherMode);
+        }
+
+        public static INamedTypeSymbol SystemNetSecurityRemoteCertificateValidationCallback(Compilation compilation)
+        {
+            return compilation.GetTypeByMetadataName(WellKnownTypeNames.SystemNetSecurityRemoteCertificateValidationCallback);
+        }
+
         #region Test Framework Types
```

Comment:
I reached up to this point in the file and then got lazy :-)

Reply:
Haha, I can chip away at this too later this weekend.

Reply:
@dotpaul Good to merge? I am planning to do another FI from master as Sam has refactored the Analyzer utilities project quite a bit, and I also want to refactor out the dataflow analysis utilities into a separate project on features/dataflow.